### PR TITLE
Document RFC84 patient and sample removal commands

### PR DESCRIPTION
**Pull request by forus**
See also [RFC84](https://docs.google.com/document/d/1bOIeLJ44r9h3m6dB390kQ_SAAn69KKxYwFTYBTy5-cI/edit) and [respective code changes in PR 46](https://github.com/cBioPortal/cbioportal-core/pull/46)

### CODE REVIEW

File: docs/Data-Loading-Maintaining-Studies.md [hunk 1]
```diff
@@ -58,3 +58,31 @@ You can also remove multiple studies at once by passing the Cancer Study Ids sep
...
+## Deleting patient information
+
+To remove a patient from a study, run:
+```
+./cbioportalImporter.py remove-patients --study_ids study1_stable_id --patient_ids patient1_stable_id,patient2_stable_id
+```
+
+You can specify multiple studies separated by comma. Here is an example of removing patient information from two studies.
+```
+./cbioportalImporter.py remove-patients --study_ids study1_stable_id,study2_stable_id --patient_ids patient1_stable_id
```
Comment (pieterlukasse) [inline docs/Data-Loading-Maintaining-Studies.md:71]: is --patient_ids restricted to one patient in this case? Or can I also have:
` --study_ids study1_stable_id,study2_stable_id --patient_ids patient1_stable_id,patient2_stable_id`? If yes, perhaps consider just adding this as the example, so it is more complete. And the sentence below would then read:
```
The patients with given stable ids have to exist in each study.
```

Comment (pieterlukasse) [reply docs/Data-Loading-Maintaining-Studies.md:71]: I like the sentence from the docker example commands .md below:
- You can specify multiple studies and/or samples. All sample ids have to exist for each specified study.

**Review by pieterlukasse — COMMENTED** [review]
thanks @forus . I added one comment that might help align the docs a bit more.

File: docs/Data-Loading-Maintaining-Studies.md (hunk 1 above)
Comment (forus) [reply docs/Data-Loading-Maintaining-Studies.md]: @pieterlukasse please have a look https://github.com/cBioPortal/cbioportal/pull/10820/commits/9fc5e028abce8d30f3c537d1e18f92a9ad584c71

**Review by pieterlukasse — APPROVED** [review]

**Pull request merged**